react/features/help/actions.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+import { SET_HELP_VIEW_VISIBLE } from './actionTypes';
+
+/**
+ * Action to set the visibility of the Help View.
+ *
+ * @param {boolean} visible - True if visible, false otherwise.
+ * @returns {{
+ *     type: SET_HELP_VIEW_VISIBLE,
+ *     visible: boolean
+ * }}
+ */
+export function setHelpViewVisible(visible: boolean) {
```

Comment:
This seems to implement the same thing dialogs do. Can't we make openDialog work here? I believe we do the same with the BottomSheet, which is also a sliding view.